wisp-launchdarkly/src/main/kotlin/wisp/launchdarkly/LaunchDarklyFeatureFlags.kt

```diff
@@ -232,6 +234,11 @@ class LaunchDarklyFeatureFlags constructor(
       return
     }
 
+    // if we're in offline mode, using the default value if ok
+    if (ldClient.isOffline) {
```

Comment:
This might be dangerous in production if the client goes offline.

Reply:
We had a chat, offline mode is specifically set on initialisation: https://docs.launchdarkly.com/sdk/features/offline-mode#java

Reply:
Reviewing docs/code, this should be fine.  Offline mode is setup at client creation.